integrationTests/chainSimulator/bridge/simpleEsdt_test.go

```diff
@@ -258,6 +233,31 @@ func createAllEsdtTypes(
 	return tokens
 }
 
+func createLocalMintArgs(
+	token string,
+	amount *big.Int,
+) string {
+	return "local_mint" +
+		"@" + hex.EncodeToString([]byte(token)) +
+		"@" +
```

Comment:
Is this ok?

Reply:
not good, not bad :smile: 
this func is only used for minting Fungible and the SC accepts `@` or `@00` for nonce argument